app/views/todos/_form.html.erb

```diff
@@ -21,6 +21,16 @@
     <% form.text_field current_user.id %>
   </div>
 
+  
+  <% unless @todo.status ==true %>
+    <div class="field">
+      <%= form.label :status %>
+      <%= @todo.status? ? "In process" : "Finish" %>
+  <% else %>
+      <%= form.check_box :estate, checked:true  %> this ToDo task is in process 
```

Comment:
This is not going to work, this variable was renamed to `status`

Reply:
true, fix it